src/app/App.jsx

```diff
@@ -18,13 +18,15 @@ import NotFound from './notFound/NotFound';
 import LoadingLabel from '../components/form/LoadingLabel';
 
 import withTracker from '../components/routing/withTracker';
+import analytics from '../services/analytics';
 
 const TeamRouter = React.lazy(() => import('./team/Router'));
 const CommentsRouter = React.lazy(() => import('./comments/Router'));
 const findRouterPromise = import('./find/Router');
 const FindRouter = React.lazy(() => findRouterPromise);
 
-history.listen((location, action) => {
+history.listen(() => {
+  analytics.track('Page View');
```

Comment:
Ah, awesome!
Can we make the event params include the URL and such, so we can know which page it is?

Reply:
Properties called "Current URL" (https://gogetta.nyc/find) and "url" (/find) are automatically added by Mixpanel :)

Reply:
Ah, great. 😄